platforms/melpomene/melpo.toml

```diff
@@ -0,0 +1,39 @@
+[kernel_cfg]
```

Comment:
nit: personally, i don't know if the `_cfg` suffix is particularly useful for these --- the whole _thing_ is a config file, so i might just call it `[kernel]`...